src/edi_energy_scraper/__init__.py

```diff
@@ -300,6 +301,25 @@ async def _download(
             raise
         return file_path
 
+    def get_edifact_format(self, path: Path) -> tuple[EdifactFormatVersion, list[EdifactFormat] | None]:
+        """
+        Determines the edifact format of a given file
```

Comment:
beschreib hier gerne den fall, dass APERAK und CONTRL in einer Datei sind. Das macht klar, warum es eine list ist und nicht nur genau 1 format.